es-core/src/utils/FileSystemUtil.cpp

```diff
@@ -138,13 +138,41 @@ namespace Utils
 
 		} // getPathList
 
+		std::string mCustomHomePath = "";
+
+		void setHomePath(std::string path)
+		{
+			mCustomHomePath = path;
+		}
+
 		std::string getHomePath()
 		{
+			if (!mCustomHomePath.empty())
+				return mCustomHomePath;
+
 			static std::string path;
 
 			// only construct the homepath once
 			if(!path.length())
 			{
+#if defined(_WIN32)
+				char buffer[MAX_PATH];
+				DWORD size = MAX_PATH;
+				DWORD result = GetModuleFileNameA(NULL, buffer, size);
+				if (result)
+				{
+					// verify if .emulationstation/es_systems.cfg is under exe's path to make app portable
+
+					std::string ret = buffer;
+					std::string portableDir = getGenericPath(getParent(ret)) + "/.emulationstation/es_systems.cfg";
```

Comment:
since your using/returning `getExePath()` below, couldn't you use that here as well?  Then this could be simplified and wouldn't need to be Windows specific.